### DISCUSSION TITLE
Normalize search for latin-based scripts

### DISCUSSION THREAD

**Discussion by ruihildt** [directus/directus#19699] [Feature Requests]
### Summary

As long as you're an English speaking country, then you mostly don't care about it. But as soon as you have to support latin-based scripts (most of Europe to start with), then it is necessary.

### Basic Example

Search for "lowenthal" will return a hit for "Löwenthal".
Search for "deja vu" will return a hit for "déjà vu".

### Motivation

Language like french commonly use accents. Users don't always know the correct spelling, and sometimes in names it's hard to know ahead if a character is accentuated or not.

For this reason, it would be very helpful to provide a way to normalize the search.

### Detailed Design

Either give an option to normalize search in the setting.

### Must Have:
- Enable normalizing the search

### Drawbacks

It has a performance cost.

### Alternatives

Provide a way to normalize through a custom resolver.

### Adoption Strategy

Just a setting to check.

**Comment by rijkvanzanten** [comment]
For what it's worth, this isn't really the case:

> As long as you're an English speaking country, then you mostly don't care about it.

There's a lot of examples even in non-accented uses where this would be appropriate 🙂 (Like searching for `Rijk’s` instead of `Rijk's`).

Should we treat this as / consider this as a bigger fuzzy search? So for example also find `Löwenthal` if you search for `Lownthal` (note the missing `e`)

**Reply by ruihildt** [reply]
Yeah fuzzy search is definitely solving it and probably even better, because typo's etc.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
